.circleci/config.yml

```diff
@@ -57,6 +57,7 @@ commands:
       - run:
           name: cargo build
           command: cargo build
+          no_output_timeout: 20m
```

Comment:
Isn't cargo build fairly chatty? 

20m of no output seems a bit long. Why are we using this value?

Reply:
It might be worth investigating ways to cut down on the syncstorage build time as part of the sync modernization work because it is getting quite lengthy

Reply:
FWIW, it appears the source of this may actually be a regression in Rust 1.61.0 itself. I've opened a PR (https://github.com/mozilla-services/syncstorage-rs/pull/1326) to address this by pinning Rust to 1.60.0. As @pjenvey noted [here](https://mozilla.slack.com/archives/CK6MA5133/p1654120513569029?thread_ts=1654114975.555839&cid=CK6MA5133), we should probably start pinning to a particular Rust version anyway (instead of just grabbing the latest version) to ensure that the version in the Dockerfile and the version in the CircleCI file always match.

Reply:
Agreed @ethowitz! Since that PR is now merged, how would you like to proceed with closing this out given the changes in the CircleCI config? Thanks!

Reply:
You can go ahead and rebase with the master branch to pull the changes in. I think we can remove this line (the one that extends the timeout to 20 minutes). From what I can tell, I think we should also remove the `rustup` commands you commented out: we're using circleci images that have the correct version of rust pre-installed, so I think those commands prob aren't necessary

Reply:
Ok, I made the changes to the file, only leaving the output of what Rust version is currently built, though is there any merit to leaving the 20 min timeout? I'm letting it run now to see how long it may take to build again given these changes.